<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="vert_Main">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lblHosts">
          <property name="text">
           <string>Hosts</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hori_Hosts">
          <item>
           <widget class="QTableWidget" name="twHosts">
            <property name="rowCount">
             <number>0</number>
            </property>
            <property name="columnCount">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="vert_HostButtons">
            <item>
             <widget class="QPushButton" name="btnAdd">
              <property name="locale">
               <locale language="English" country="UnitedKingdom"/>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnEdit">
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnRemove">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vert_Notifiyng">
       <item>
        <layout class="QHBoxLayout" name="hori_AutoScroll">
         <item>
          <widget class="QCheckBox" name="chkScroll">
           <property name="text">
            <string>AutoScroll</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hori_LogLevel">
         <item>
          <widget class="QLabel" name="lblLogLevel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>LogLevel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hSpc1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="cmbLogLevel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>Emergence</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Alert</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Critical</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Error</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Warning</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Notice</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Information</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Debug</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DebugEx</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hori_Notifier">
         <item>
          <widget class="QLabel" name="lblNotifier">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Notifier</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hSpc2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="cmbNotifier">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>Emergence</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Alert</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Critical</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Error</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Warning</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Notice</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Information</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Debug</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DebugEx</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hori_NotifierChecks">
         <item>
          <widget class="QCheckBox" name="chkNotifier">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Enabled</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hSpc3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="radioTray">
           <property name="text">
            <string>Tray</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioSystem">
           <property name="text">
            <string>System</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
